server/routerlicious/kubernetes/routerlicious/values.yaml

```diff
@@ -19,15 +19,39 @@ alfred:
   numberOfMessagesPerTrace: 100
   sessionStickinessDurationMs: 3600000
   throttling:
-    restCalls:
+    restCallsPerTenant:
         maxPerMs: 1000000
         maxBurst: 1000000
         minCooldownIntervalInMs: 1000000
         minThrottleIntervalInMs: 1000000
         maxInMemoryCacheSize: 1000
         maxInMemoryCacheAgeInMs: 60000
         enableEnhancedTelemetry: false
-    socketConnections:
+    restCallsCreateDoc:
```

Comment:
the naming is a little inconsistent here. For some values we are using pertenant and percluster, but for some we are not.
Let make it consistent?
you can also do something like:
restCallsPerCluster:
  createDoc: <>
   GetDeltas: <>

Reply:
Add the percluster postfix afterward, and I didn't adjust the current config structure to avoid the potential bug.

Reply:
Sounds good! Adjusted the naming convention.